### DISCUSSION TITLE
Hello

### DISCUSSION THREAD

**Discussion by Phoenix-rat** [community/community#7495] [Copilot Conversations]
I am using Atom for code. But Github Copilot isnt support Atom. How I can Install Github Copilot to Atom?

**Comment by Arhell** [comment]
https://copilot.github.com/#faq-what-development-environments-are-supported

**Reply by Jim-Salmons** [reply]
Hi @Phoenix-rat,

As @Arhell cogently stated via link, Copilot is only available on Microsoft's VS Code and JetBrains' PyCharm/IntelliJ IDEs. So for the meantime, you can explore Copilot on either of these IDEs and simply pop back and forth interactively from one of them to Atom as you like.  VS Code is more lightweight than the JetBrains toolset and both companies offer free Community Editions of these tools.

I use PyCharm Pro which is working great for my research development and I am fairly confident the Copilot plugin can be installed in the Community Edition. While I know you may be disappointed that Copilot is not available for Atom at this time, I strongly encourage you to try it by installing one of its free supported platforms. I am only just getting started with my own use (as a disabled developer) and Copilot was shown itself to be truly transformative for my daily use.

Welcome to the community.

Happy Healthy Vibes from Colorado USA,
  Jim
